rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/common/SchedulerRestInterface.java

```diff
@@ -2413,4 +2413,41 @@ JobValidationData validateJobSignal(@HeaderParam("sessionid") String sessionId,
     @Produces(MediaType.APPLICATION_JSON)
     Map<String, Map<String, Boolean>> checkJobsPermissionMethods(@HeaderParam("sessionid") String sessionId,
             PermissionForm multipart) throws RestException;
+
+    /**
+     *
+     * Add an external endpoint url to the endpoint list of the considered job
```

Comment:
Add an external endpoint url to a job. The job must be alive and not terminated, otherwise an UnknownJobException will be thrown.

Reply:
Ok